src/main/java/org/openhab/binding/zwave/handler/ZWaveThingHandler.java

```diff
@@ -130,13 +130,17 @@ public void initialize() {
 
         final BigDecimal cfgNodeId = (BigDecimal) getConfig().get(ZWaveBindingConstants.CONFIGURATION_NODEID);
         if (cfgNodeId == null) {
-            logger.error("NodeID is not set in {}", getThing().getUID());
+            logger.debug("NodeID is not set in {}", getThing().getUID());
```

Comment:
Probably this should be WARN - this is quite a serious error and the thing will not work.

Reply:
The user will be informed through the thing status.